Source/Extensions/Blazorise.DataGrid/DataGridDateColumn.cs

```diff
@@ -10,4 +13,64 @@ public class DataGridDateColumn<TItem> : DataGridColumn<TItem>
     /// Hints at the type of data that might be entered by the user while editing the element or its contents.
     /// </summary>
     [Parameter] public DateInputMode InputMode { get; set; }
+
+    /// <summary>
+    /// Gets the date format based on the current <see cref="InputMode"/> settings.
+    /// </summary>
+    [Parameter] public string DateDisplayFormat { get; set; }
```

Comment:
Can we reuse the `DisplayFormat` from `BaseDataGridColumn`? It is the same thing.

Reply:
Makes sense, although this would have the advantage that you could have a different display format (when viewing the table) and edit format (when editing the records)